src/dbnode/persist/fs/bloom_filter.go

```diff
@@ -74,15 +78,17 @@ func newManagedConcurrentBloomFilterFromFile(
 	numElementsM uint,
 	numHashesK uint,
 	forceMmapMemory bool,
+	reporterOptions mmap.ReporterOptions,
```

Comment:
Might be better to just propagate the `mmap.Reporter` then set the `ReporterOptions` here ourselves?

That way it's pretty clear and in control of setting up the `Context` itself, etc. Also not clear if it needs to add tags if needs to check if `Context` already has tags and to take a copy and modify, etc. Just seems cleaner to construct `ReporterOptions` and `Context` with the given reporter right before use.